### ISSUE TITLE
Multiline HTML comments break the table layout

### ISSUE THREAD

**Issue by runarberg** [HcySunYang/vuese#120]
**Describe the bug**
If you leave an empty line in an HTML commit (like the once before a slot) it will break the table layout for the slots section.

**To Reproduce**
Create a component with a slot and document it using an empty line:

```vue
<template>
  <div>
    <!--
      Form header

      This is another paragraph.
    -->
    <slot name="header"></slot>
  </div>
</template>
```

**Expected behavior**
Preferably the cell describing the slot should have two paragraphs in it. But trimming of the white space is fine as well.

**Actual behavior**
The second paragraph gets a code indentation, and all the subsequent slots will have weird markup.

**Screenshots**
![image](https://user-images.githubusercontent.com/3984600/68640515-e58ff100-04bc-11ea-910a-5bc36577add1.png)

**Environmental information:**
```
Environment Info:

  System:
    OS: Linux 5.3 Ubuntu 19.10 (Eoan Ermine)
    CPU: (4) x64 Intel(R) Core(TM) i7-7600U CPU @ 2.80GHz
  Binaries:
    Node: 12.12.0 - ~/.nvm/versions/node/v12.12.0/bin/node
    Yarn: 1.19.1 - ~/work/stemson/ui/node_modules/.bin/yarn
    npm: 6.11.3 - ~/.nvm/versions/node/v12.12.0/bin/npm
  Browsers:
    Firefox: 70.0.1
```

**Comment by QuinnBast** [comment]
Can confirm this with multi-line comments inside of a table cell as well. To get any code in a table cell you have to use the single back-tick and cannot use the triple backtick in the form of a code block. This might be its own issue though.

![image](https://user-images.githubusercontent.com/22599103/73005490-a9d3aa80-3dce-11ea-8aec-26925d9113f3.png)

**Comment by adjenks** [comment]
Yeah html comments break markdown rendering. When `<!-- something-->` is added right above. a markdown table, the table does not render at all.
